### DISCUSSION TITLE
Fusion FormTabs and FormSteps

### DISCUSSION THREAD

**Discussion by bruno-darochac** [vueform/vueform#186] [Questions]
Hello I wanted to know if it was possible to fusion FormTabs and FormSteps.

I want to have FormTabs to selection different tabs and into it have FormSteps to for example register user with lot of information.. 

Or if you have any alternative for nice tabs it will be a pleasure to have your advice :)

Thanks in advance :)

**Comment by adamberecz** [comment]
Unfortunately only either Vueform steps or tabs can be used for a form.
